test/component_test.go

```
@@ -0,0 +1,218 @@
+	defer clientset.NetworkingV1().Ingresses(ingressNamespace).Delete(context.Background(), ingressName, metav1.DeleteOptions{})
+	_, err = clientset.NetworkingV1().Ingresses(ingressNamespace).Create(context.Background(), ingress, metav1.CreateOptions{})
+	assert.NoError(s.T(), err)
+	assert.NotNil(s.T(), ingress)
+
+	// Wait for the Ingress to be updated with the LoadBalancer metadata
+	factory := informers.NewSharedInformerFactory(clientset, 0)
+	informer := factory.Networking().V1().Ingresses().Informer()
+
+	stopChannel := make(chan struct{})
+
+	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

golangci-lint (1.64.8)

Error return value of `informer.AddEventHandler` is not checked

(errcheck)